src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -387,4 +395,8 @@
     text: |
         az keyvault update --enabled-for-disk-encryption true --name MyKeyVault --resource-group MyResourceGroup
     crafted: true
+  - name: Update the properties of a key vault. (autogenerated)
+    text: |
+        az keyvault update --enable-soft-delete true --name MyKeyVault
+    crafted: true
```

Comment:
This example is not necessary since `--enable-soft-delete`  will be deprecated soon.

Reply:
It's a good question that we show whether show the deprecating parameter or not.